flink-connectors/flink-connector-aws-base/src/main/java/org/apache/flink/connector/aws/util/AWSGeneralUtil.java

```diff
@@ -273,7 +274,15 @@ public static SdkAsyncHttpClient createAsyncHttpClient(
 
     public static SdkAsyncHttpClient createAsyncHttpClient(
             final AttributeMap config, final NettyNioAsyncHttpClient.Builder httpClientBuilder) {
+        return createAsyncHttpClient(config, httpClientBuilder, SdkEventLoopGroup.builder());
```

Comment:
> SdkEventLoopGroup.builder()

If anyone is interested what the default behaviour is, because I was, then here is a link:
- https://github.com/aws/aws-sdk-java-v2/blob/master/http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/NettyNioAsyncHttpClient.java#L148

It will use `SharedSdkEventLoopGroup::get`